html/php-pre-challenge2/index.php

```diff
@@ -3,7 +3,13 @@
 
 // 修正はここから
 for ($i = 0; $i < count($array); $i++) {
```

Comment:
こちらもともとの記述ですが、無駄な処理が多いので修正してみてください。

```suggestion
$size = count($array);
for ($i = 0; $i < $size; $i++) {
```

for文はかきのように動きます。

```
①$i=0;を実行
②$i < count($array); の比較を実行
③for文の中の処理を行う
④$i++を実行
⑤②の比較を行う。trueなら、引き続き処理を行う。以下、同じ。
```

もともとのコードですと、②や⑤の比較が毎回行われてしまうので無駄になってしまいます。
（ついでにいうと、2つめのforループでも、`count($array)`がなんども呼び出されています）。

`count($array)`の値は、常に同じ（呼び出しごとに変わらない）ので、別途変数に切り出したほうがパフォーマンスが良いです。
count程度の処理は軽いのですが、きちんとfor文の動きを知っておくほうが良いと思いの指摘です。
※ 今回の実装では、count($array)を 13^2回 呼び出してると思います。